static/app/views/integrationPipeline/awsLambdaFunctionSelect.tsx

```diff
@@ -72,8 +71,12 @@ export default class AwsLambdaFunctionSelect extends Component<Props, State> {
     return Object.values(this.model.getData()).every(val => val);
   }
 
+  get formFields() {
+    const data = this.model.getTransformedData();
+    return Object.keys(data).map(key => ({name: key, value: data[key]}));
```

Comment:
nit slightly more terse

```suggestion
    return Object.entries(data).map(([name, value]) => ({name, value}));
```